bugbug/models/stepstoreproduce.py

```diff
@@ -62,21 +63,23 @@ def __init__(self, lemmatization=False):
         self.clf.set_params(predictor="cpu_predictor")
 
     def get_labels(self):
+        defects = DefectModel.get_bugbug_labels(self, "bug")
```

Comment:
I think you might be able to just use the `bug_type` field, checking `bug["bug_type"] == "defect"` instead of `bug["id"] in defects`.
Can you check how many bugs you'd have in the training set in the first vs the second case?

Reply:
Do i need to train defect model first? because there wasn't a single bug with 'bug_type' field when i ran this.

Reply:
Sorry, the field name is actually "type", not "bug_type".

Reply:
(And make sure you have the latest bugs DB)

Reply:
It worked! The dataset actually increased a bit, the comment above contains the current result.